docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,36 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
+
+1.  Install the operator.
+
+    1. Download the manifest from GitHub.
+
+        ```
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/deploy/installer/generated/host-definition.yaml > host-definition.yaml
+        ```
+
+    2. Verify that the hostdefinition is in the _Ready_ phase.
```

Comment:
@matancarmeli7 how could this be ready if no yaml was applied?

Reply:
changed

Reply:
I still don't see any "apply" (like e.g. [step 2.c.](https://www.ibm.com/docs/en/stg-block-csi-driver/1.9.0?topic=driver-installing-github))

Reply:
added

Reply:
thanks, please write here once it's available for review

Reply:
available